example/cavity/Solid/previous_work

```diff
@@ -0,0 +1,4952 @@
+0.22000000000000064 -0.000012460392149638011
```

Comment:
What is this file about? The name `previous_work` is confusing. Is this a precomputed initial solution?